### PR TITLE
mech tweaks

### PR DESCRIPTION
**Pull request by deltanedas**
## About the PR
BaseMech and BaseMechPart(Item) referenced ripley too much. removed them, and for parts, put them in their own ripley base part prototypes.
also did a little cleanup on RipleyMech - typo and duplicate components in batteries included variant

**Media**
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
no cl no fun

### CODE REVIEW

**Comment by deltanedas** [comment]
@EmoGarbage404 i think you made mechs right so is everything good here

**Comment by deltanedas** [comment]
also uh im gonna put starting battery into its own thing since having it in mech is very silly

**Comment by deltanedas** [comment]
removed startingBattery and using container fill instead, everything works as normal

File: Content.Server/Mech/Systems/MechSystem.cs [hunk 1]
```diff
@@ -93,6 +96,18 @@ private void OnInteractUsing(EntityUid uid, MechComponent component, InteractUsi
         }
     }
 
+    private void OnInsertBattery(EntityUid uid, MechComponent component, EntInsertedIntoContainerMessage args)
+    {
+        if (!TryComp<BatteryComponent>(args.Entity, out var battery))
```
Comment (EmoGarbage404) [inline Content.Server/Mech/Systems/MechSystem.cs:101]: check that the container is actually the batter slot container

Comment (deltanedas) [reply Content.Server/Mech/Systems/MechSystem.cs:101]: why would you make a powercell mech equipment tho...

**Comment by deltanedas** [comment]
ok you wanted it :trollface:

**Comment by Emisse** [comment]
needs emo approval for merge

**Review by EmoGarbage404 — APPROVED** [review]
you've received it

**Pull request merged**